css/styles.css

```diff
@@ -27,6 +20,6 @@ li {
   display: block;
   padding: 0.25rem;
```

Comment:
you could use px for padding here as the value is small, otherwise looks great

Reply:
Will update it. Thanks